### PR TITLE
Remove RGS

### LINKED ISSUE #2014: Remove rgs sync
Unnecessary overhead which is polluting the logs.

### PR DESCRIPTION
**Pull request by luckysori**
Fixes #2014.

### CODE REVIEW

File: crates/ln-dlc-node/src/tests/load.rs [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+mod coordinator;
+
+const ESPLORA_ORIGIN_PUBLIC_REGTEST: &str = "http://34.32.0.52:3000";
+const ORACLE_ORIGIN_PUBLIC_REGTEST: &str = "http://34.32.0.52:8081";
+const ORACLE_PUBKEY_PUBLIC_REGTEST: &str =
+    "5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859";
+
+#[tokio::test(flavor = "multi_thread")]
+async fn single_app_many_positions_load() {
```
Comment (bonomat) [inline crates/ln-dlc-node/src/tests/load.rs:30]: What's this test?

Comment (bonomat) [reply crates/ln-dlc-node/src/tests/load.rs:30]: Oh, because it was a git revert I guess... maybe this needs manual intervention

File: maker/src/bin/maker.rs [hunk 2]
```diff
@@ -0,0 +1,230 @@
+use anyhow::Context;
```
Comment (bonomat) [inline maker/src/bin/maker.rs:1]: Lol, definitely needs some manual editing... we've deleted this project 😅

File: coordinator/Cargo.toml [hunk 3]
```diff
@@ -26,7 +26,7 @@ serde_json = "1"
 sled = "0.34"
 thiserror = "1.0"
 tokio-metrics = "0.2.2"
-toml = "0.8"
+toml = "0.7.3"
```
[the remark this answers is not in the document]
Comment (luckysori) [reply coordinator/Cargo.toml:29]: Good catch!

**Comment by luckysori** [comment]
Thanks for the review, @bonomat. I just pushed it as soon as it compiled haha.

File: .github/workflows/android-fastlane.yml [hunk 4]
```diff
@@ -154,12 +146,12 @@ jobs:
         run: |
           echo "$FASTLANE_ANDROID_SERVICE_ACCOUNT_KEY_BASE64" | base64 --decode > ${{ runner.temp }}/fastlane_service_account.json
           echo "$FASTLANE_ANDROID_UPLOAD_KEY_BASE64" | base64 --decode > ${{ runner.temp }}/upload-keystore.jks
-
+
```
Comment (holzeis) [inline .github/workflows/android-fastlane.yml:149]: Was that change intentional?

**Review by holzeis — APPROVED** [review]
LGTM, thank you 👍

**Review by bonomat — APPROVED** [review]
LGTM

File: .github/workflows/android-fastlane.yml (hunk 4 above)
Comment (luckysori) [reply .github/workflows/android-fastlane.yml:149]: I think Emacs just "fixed" it.

**Comment by luckysori** [comment]
Superseded by #2094.
